src/commands/link.ts

```diff
@@ -63,6 +70,14 @@ export const call = async (interaction: CommandInteraction) => {
   const linkInfo = LINKS[topic];
   let replyMsg = `See ${hyperlink(linkInfo.description, hideLinkEmbed(linkInfo.url))}`;
   if (target) replyMsg = `${userMention(target.id)} ${replyMsg}`;
+  else
+    replyMsg = `${replyMsg}
+
+${italic(
+  `Note: to use this command in response to a user's query, specify the ${inlineCode(
+    "target"
+  )} option when invoking the command.`
+)}`;
```

Comment:
When the code looks ugly like this, it's usually a sign to extract it out. I'd move `if (target)` condition to a separate function.

```typescript
  const msg = `See ${hyperlink(linkInfo.description, hideLinkEmbed(linkInfo.url))}`;
  await interaction.reply({
    content: maybeMention(msg, target),
    ephemeral: !target,
  });
```